R/fit_mmrm.R

```diff
@@ -79,10 +76,6 @@ h_get_diagnostics <- function(fit) {
 #'   - `heterogeneous compound symmetry`: Heterogeneous Compound Symmetry covariance matrix, which uses
 #'        `T + 1` variance parameters.
 #'
-#'   For the `optimizer`, the user can choose among alternatives to the recommended `"automatic"`,
-#'   please see [mmrm::refit_multiple_optimizers()] for details. Usually it should not be necessary
-#'   to use a specific optimizer and the `"automatic"` setting should be kept.
-#'
```

Comment:
is this information coming from `mmrm::mmrm()` documentation? if yes I think it is fine to reference it as it is done above (`@param ... additional arguments for [etc..`)

Reply:
Otherwise, it could be added back into `@details` as it is done for `method`